### PR TITLE
Add possibility to remove tinc tunnel

### PR DESCRIPTION
**Pull request by fredreichbier**
Working on #21

### CODE REVIEW

File: authappliance/menu.py [hunk 1]
```diff
@@ -465,6 +467,34 @@ def setup_tinc(self, local_vpn_ip, remote_vpn_ip, vpn_subnet, vpn_name='privacyi
...
 
+    def is_tinc_configured(self, vpn_name='privacyideaVPN'):
+        """ Return true if /etc/tinc/privacyideaVPN exists and /etc/tinc/nets.boot contains the VPN name """
+        if os.path.exists(os.path.join('/etc/tinc', vpn_name)):
+            nets_boot = NetsBoot(LocalIOHandler(), '/etc/tinc/nets.boot')
+            if vpn_name in nets_boot.nets:
+                return True
+        return False
+
+    def stop_tinc(self, vpn_name='privacyideaVPN'):
```
Comment (cornelinux) [inline authappliance/menu.py]: This method name might be misleading.
Is there a way to restart tinc anyways?

"stop_tinc" sounds like stopping the tunnel but not deleting the configuration.
Should there be a parameter, like "also_delete_the_tunnel_definition=False"?

Maybe I would simply rename the function to "delete_tinc" (this is what it does)

Comment (cornelinux) [reply authappliance/menu.py]: OK - it is also called "stop_redundancy".

Question: If a redundancy is decommissioned and tinc is not deleted. What happens if I set up a new redundancy?
SHouldn't we delete the tinc anyways, if we delete the redundancy?

Comment (fredreichbier) [reply authappliance/menu.py]: Hm, you're right. We should probably just delete the tinc VPN anyways when deleting redundancy.

Or, alternatively, we could check whether a tinc tunnel named ``privacyideaVPN`` already exists when setting up redundancy?

Comment (fredreichbier) [reply authappliance/menu.py]: fa7a00a now removes the tinc tunnel when removing redundancy.

**Comment by cornelinux** [comment]
Hm, now it is up to me to resolve the conflicts? ;-)

**Pull request merged**